### PR TITLE
Add some style to the button

### PR DESCRIPTION
**Pull request by jcarrey**

### CODE REVIEW

**Comment by jcarrey** [comment]
image

**Review by AngelZartis — APPROVED** [review]

File: factorial-clockin-copier/main.js [hunk 1]
```diff
@@ -147,6 +147,15 @@
     const element = filledRowElement.querySelector("td:nth-child(2)");
     const el = document.createElement("button");
     el.innerText = "Copy on empty days";
+    Object.assign(el.style, {
```
Comment (GonArriviComparis) [inline factorial-clockin-copier/main.js:150]: Why Object.assign over a simple assignation?
`el.style = {...}`

File: factorial-clockin-copier/main.js [hunk 2]
```diff
@@ -147,6 +147,15 @@
...
+      backgroundColor: '#C673B3',
+      padding: '8px',
+      border: '1px solid #a91f8954',
+      'border-radius': '5px',
```
Comment (GonArriviComparis) [inline factorial-clockin-copier/main.js:154]: Why not continue with camelcase properties notation?

**Review by GonArriviComparis — APPROVED** [review]
I think everything is sound, just added a couple of questions to know if there is a deeper meaning behind some choices.

File: factorial-clockin-copier/main.js (hunk 1 above)
Comment (jcarrey) [reply factorial-clockin-copier/main.js]: Not overriding existing styles, there is none, so it should be fine either way.

**Pull request merged**